qa/tasks/cephfs/test_mds_metrics.py

```diff
@@ -489,25 +494,37 @@ def test_client_metrics_and_metadata(self):
         self.mount_a.create_files()
 
         #creating another filesystem
-        fs_b = self._setup_fs(fs_name = "fs2")
+        fs_b = self._setup_fs(fs_name="fs2")
 
         # Mount a client on fs_b
         self.mount_b.mount_wait(cephfs_name=fs_b.name)
         self.mount_b.write_n_mb("test.bin", 1)
         self.mount_b.path_to_ino("test.bin")
         self.mount_b.create_files()
 
+        fscid_list = [fs_a.id, fs_b.id]
+
+        # wait a bit for mgr to get updated metrics
+        time.sleep(5)
+
         # validate
         valid, metrics = self._get_metrics(
-            self.verify_mds_metrics(client_count=TestMDSMetrics.CLIENTS_REQUIRED), 30)
+            self.verify_mds_metrics(client_count=1, mul_fs=fscid_list), 30)
```

Comment:
Why `client_count` is reduced here?

Reply:
`test_client_metrics_and_metadata` checks only the metrics are properly updated or not which can be checked by having only one client too, no need to have extra clients.